bin/setup

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'fileutils'
+include FileUtils
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
```

Comment:
Printing out the args like this will print it out like an Array...you probably want something like `Command #{args.join(" ")} failed`

Reply:
:+1: I pulled this from the swagger generated `bin/setup` in [topological_inventory-api/bin/setup](https://github.com/ManageIQ/topological_inventory-api/blob/master/bin/setup#L10) I'll push a PR to fix it there as well.